studio/src/router.js

```diff
@@ -163,6 +172,7 @@ export class Router extends EventTarget {
         this.linkStoreToHash(Store.sort, ['sortBy', 'sortDirection'], getSortDefaultValue);
         this.linkStoreToHash(Store.placeholders.search, 'search');
         this.linkStoreToHash(Store.commerceEnv, 'commerce.env', WCS_ENV_PROD);
```

Comment:
should we delete/conment: `this.linkStoreToHash(Store.commerceEnv, 'commerce.env', WCS_ENV_PROD);` ?